Neos.Neos/Classes/Controller/Backend/ContentController.php

```diff
@@ -147,12 +146,9 @@ public function initializeUploadAssetAction(): void
     public function uploadAssetAction(Asset $asset, string $metadata, string $node, string $propertyName)
     {
         $nodeAddressString = $node;
```

Comment:
$nodeAddressString is only assigned to pass it on the $nodeAddress, perhaps we can just do 
$nodeAddress = NodeAddress::fromJsonString($node);